#
# Submitted-By: Marc Kleine-Budde, 2006-02-09
# Committed-By: Marc Kleine-Budde
#
# Error:
#
# libpng-config is not SYSROOT capable,
#
# Description:
#
# libpng-config spits out path like /usr/lib, this patch teaches to
# honour SYSROOT
#
# State:
#
# not submittet upstream, (shame on me)
#
Index: libpng-1.2.8-config/Makefile.am
===================================================================
--- libpng-1.2.8-config.orig/Makefile.am
+++ libpng-1.2.8-config/Makefile.am
@@ -55,9 +55,9 @@ uninstall-local:
 
 libpng-config:
 	( cat ${srcdir}/scripts/libpng-config-head.in; \
-	echo prefix=\"$(prefix)\"; \
-	echo I_opts=\"-I$(includedir)\"; \
-	echo L_opts=\"-L$(libdir)\"; \
+	echo prefix=\"\$$SYSROOT$(prefix)\"; \
+	echo I_opts=\"-I\$$SYSROOT$(includedir)\"; \
+	echo L_opts=\"-L\$$SYSROOT$(libdir)\"; \
 	echo R_opts=\"-Wl,-rpath,$(libdir)\"; \
 	echo libs=\"-lpng12 -lz -lm\"; \
 	cat ${srcdir}/scripts/libpng-config-body.in ) > libpng-config
